src/side-loaded/basic-sideload/sideloaded.run.ts

```diff
@@ -0,0 +1,148 @@
+import { equal } from 'node:assert';
+import {
+  Field,
+  Mina,
+  Bytes,
+  Crypto,
+  createEcdsa,
+  createForeignCurve,
+  Hash,
+} from 'o1js';
+
+import {
+  EcdsaProgram,
+  KeccakProgram,
+  CounterProgram,
+  generateEcdsaProof,
+  generateKeccakProof,
+  generateCounterProof,
+} from './sideloaded-zkprograms.js';
+
+import { MyFavoriteZkApp } from './verifier-zkapp.js';
```

Comment:
nit: same comment about a more descriptive contract name to reduce cognitive over head for first time readers.